README.md

```diff
@@ -6,7 +6,7 @@ Fabric mod where anyone can PR anything, concerning or not (no NSFW content). I'
 ## How can I ~~fuck up~~ contribute to this project ?
 
 1) Create what you want : you can do ANYTHING , adding blocks, items, whatever you want, mess with mixins, break vanilla mecanics, add dependencies, PR entire projects, as soon as it doesn't break other contributions and that it doesn't include any kind of virus, keylogger or backdoor. Game hacks are fine, system hacks are not.
-2) If possible, make it configurable. Not everyone wants to have a completly messed up rendering or a "concern" block that generates everywhere
```

Comment:
this is a very controversial change. i don't believe it'll make it past reviewing phase

Reply:
no, bc people generally add features that tend to break things/the entire game, so its good to remind people to make things optiionals